chapter3/exercise1.js

```diff
@@ -0,0 +1,30 @@
+(function() {
+
+   var min = function(firstNumber, secondNumber) {
+        var result;
+
+        if((typeof firstNumber == "undefined") || (typeof secondNumber == "undefined")) {
+            result = "You don't pass two needed arguments to function.";
+            return console.log(result);
```

Comment:
zwracaj wynik a nie konsol.loga

Reply:
zgadzam się, mnie też to się nie spodobało. :)